src/ontology/ro-edit.owl

```diff
@@ -44,6 +44,7 @@ Declaration(Class(obo:BFO_0000031))
 Declaration(Class(obo:CHEBI_50906))
 Declaration(Class(obo:COB_0001000))
 Declaration(Class(obo:OGMS_0000031))
+Declaration(Class(obo:PATO_0001470))
```

Comment:
Why this PATO term was added in the edit file?

Reply:
Same as other pull request.  This must have come in from a refresh to update from master branch.

Reply:
Ok, I'll remove it.